### ISSUE TITLE
Exact naming of the ispell dictionaries matters

### ISSUE THREAD

**Issue by msievers** [metaminded/etikett#4]
tl;dr;

Lowercase filenames of the ispell dictionaires, because postgres does not find them with uppercased parts. Tested on ubuntu 14.04.

Details

In `etikett/db/migrate` there are two dictionary files named
- `de_DE_frami.affix`
- `de_DE_frami.dict`

The corresponding migration which depends on these files beeing copied to `#{pg_config --sharedir}" has this sql statement

`execute("CREATE TEXT SEARCH DICTIONARY german_ispell (TEMPLATE = ispell, dictfile = de_DE_frami,afffile = de_DE_frami,StopWords = german);")`

Issueing this command with `psql` on a database named foo gives an error, saying the the dictionary files cannot be found, although they are in place.
